### DISCUSSION TITLE
שאלה 3 בחלק ג מועד א 2023 תלכ"צ

### DISCUSSION THREAD

**Discussion by jammaker** [NuclearGandhi/technion_second_brain#12] [טעויות ותיקונים]
הסיבה שיש הבדל בין התשובה שלך לבין התשובה בבחינה היא ששכחת להתחשב בנצילות המכונה (ביקשו את ההספק החשמלי ולא את ההספק של החיתוך).
בגלל שהנצילות היא 0.8, התשובה הנכונה היא:
P/0.8=8,488/0.8=10,610

**Comment by NuclearGandhi** [comment]
הנצילות הייתה נתונה? אני לא מוצא את זה

**Comment by jammaker** [comment]
![Screenshot_20240825_142004_Drive](https://github.com/user-attachments/assets/887c40ca-e049-4937-aaf0-a76944f33db1)
זה מהשאלה המקורית

**Reply by jammaker** [reply]
מתברר שמי שכתב את השחזור שכח להוסיף אץ הנתון הזה, אבל הוא היה בבחינה המקורית

**Reply by NuclearGandhi** [reply]
אשכרה, תודה. מוסיף את ההערה.
